examples/helloworld/README.rst

```diff
@@ -0,0 +1,75 @@
+==================
+Helloworld Example
+==================
+
+**Requires: python3**
+
+This example creates a two activities and one decider for those activities. The first activity will
+ask for a name and then the second activity will print that name.
+
+How To
+------
+
+Preconfiguration
+~~~~~~~~~~~~~~~~
+
+First an IAM user with privileges in SWF is needed. Can inline policy for an existing user can be:
```

Comment:
Please refer to http://docs.aws.amazon.com/amazonswf/latest/developerguide/swf-dev-iam.html for more details, but the inline policy for an existing user could be: